carma_wm_ctrl/include/carma_wm_ctrl/WMBroadcaster.h

```diff
@@ -440,6 +461,11 @@ class WMBroadcaster
   size_t update_count_ = 0; // Records the total number of sent map updates. Used as the set value for update.seq_id
 
   carma_wm::SignalizedIntersectionManager sim_;
+  const uint8_t acknowledgement_status__acknowledged = 1;
+  const uint8_t acknowledgement_status__rejected = 2;
```

Comment:
Are these two const effectively an enum?
Can it be reworked to ?
```
enum class AcknowledgementStatus {
  ACKNOWLEDGED=1
  REJECTED=2
}
```
Just good practice for this kind of thing